src/webgl/p5.Geometry.js

```diff
@@ -238,12 +240,16 @@ p5.Geometry.prototype._makeTriangleEdges = function() {
  * @chainable
  */
 p5.Geometry.prototype._edgesToVertices = function() {
+  const data = this.lineVertexColors.slice();
+  this.lineVertexColors.length = 0;
   this.lineVertices.length = 0;
   this.lineNormals.length = 0;
 
   for (let i = 0; i < this.edges.length; i++) {
-    const begin = this.vertices[this.edges[i][0]];
-    const end = this.vertices[this.edges[i][1]];
+    const e0 = this.edges[i][0];
```

Comment:
It's a little more verbose but maybe we can call these `endIndex0` and `endIndex1` to be clearer what they represent

Reply:
It's certainly easier to understand... I think it's better to read and understand even if it's verbose, so I'm going to do it.